src/taco-kits/TacoKitMetadata.json

```diff
@@ -281,6 +279,90 @@
                     "version": "1.1.0"
                 }
             }
+        },
+        "tacokit@3.0.0": {
+            "cordova-cli": "5.4.1",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=722537",
+            "default": true,
+            "platforms": {
+                "android": {
+                    "version": "4.1.1"
+                },
+                "ios": {
+                    "version": "3.9.0"
+                },
+                "windows": {
+                    "version": "4.0.0"
+                },
+                "wp8": {
+                    "version": "3.8.1"
+                }
```

Comment:
Is this a conscious decision? Cordova 5.4.1 seems to pin Windows 4.1.0 and Wp8 3.8.2.